policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/first_nations_tax/rate.yaml

```diff
@@ -0,0 +1,11 @@
+description: Yukon shifts this portion of the income tax paid by residents of settlement land to First Nations.
+values:
+  2022-01-01: 0.95
+metadata:
+  unit: /1
+  label: Yukon First Nations Tax Rate
+  reference:
+  - title: Government of Canada - Yukon Information Guide
```

Comment:
This reference doesn't mention 0.95. Please only list references that back up the entered values.